start.bat

```diff
@@ -0,0 +1,4 @@
+set OPENAI_API_KEY=sk-111111111111111111111111111111111111111111111111
+set OPENAI_API_BASE=http://127.0.0.1:5000/v1
```

Comment:
Let's remove this file and instead include instructions for setting OPENAI_API_BASE when using a local model in the README.